### DISCUSSION TITLE
student dev pack github

### DISCUSSION THREAD

**Discussion by subhammishra7123** [community/community#88417] [GitHub Education]
### Hi everyone,

yesterday i got verified but still now i had not gt my dev pack what to do

**Comment by rizmyabdulla** [comment]
It may take some time for the student developer pack benefits to active, so please be patient for at least one week.

**Comment by kanduru-abhiram** [comment]
Hi @subhammishra7123, please check whether your Application request is approved and the background color of Approved in green.

![image](https://github.com/community/community/assets/85011280/243cdc64-5fc7-4443-aeac-c36224d65c0b)

And wait till the background color of Approved turns to Violet, once turned, you'll get the access to all the benefits of Student Developer pack.

![image](https://github.com/community/community/assets/85011280/8809cebe-d954-4843-9e2f-4ef38cc5b082)

Please accept this as a solution if helped, thanks.
